shakedown/dcos/service.py

```diff
@@ -225,6 +252,28 @@ def wait_for_mesos_task_removal(task_name, timeout_sec=120):
     return time_wait(lambda: mesos_task_not_present_predicate(task_name), timeout_seconds=timeout_sec)
 
 
+def delete_persistent_data(role, principal, zk_node):
```

Comment:
The intent is to have something set up for when DC/OS supports deleting volumes natively (1.10 timeframe IIRC?), at which point this can be switched over to the new support without tests needing to know about the change. See `service.uninstall_package_and_data()` which invokes this.